eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -68,7 +79,7 @@ jobs:
           --logger "trx;LogFileName=$(TestTargetFramework).trx" --logger:"console;verbosity=normal"
           --blame-crash-dump-type full --blame-hang-dump-type full --blame-hang-timeout ${{parameters.TestTimeoutInMinutes}}minutes
           /p:SDKType=${{ parameters.SDKType }}
-          /p:ServiceDirectory=${{ parameters.ServiceToTest }}
+          /p:ServiceDirectory=${{ parameters.ServiceDirectory }}
           /p:IncludeSrc=false /p:IncludeSamples=false /p:IncludePerf=false /p:IncludeStress=false
```

Comment:
You should be passing in the `/p:ProjectListOverrideFile=${{ parameters.ProjectListOverrideFilePropertyName }}`